### DISCUSSION TITLE
item-filter

### DISCUSSION THREAD

**Discussion by TBCNoob** [Lailloken/Exile-UI#353] [Q&A]
I am a complete noob when it comes to this kind of stuff. I want to create custom item color drop. I am not sure where to create the edits.

out of topic, how do I know if the UI is actually working or not, I do have the cog thing and I played around with it a little. If anyone is willing to jump on call w me and explain it a little it would be much appreciated.

**Comment by Lailloken** [comment]
This discussion section isn't very active, aside from me answering user-specific questions, so you probably won't get many replies.

> how do I know if the UI is actually working or not, I do have the cog thing and I played around with it a little

generally, you'd pick the features that seem most useful to you and then enable them. You can check the wiki on how to use them correctly.

> I want to create custom item color drop. I am not sure where to create the edits.

for more general stuff about PoE outside my tool, I'd suggest checking youtube tutorials or reddit
